finalPrep/modZeroHero.js

```diff
@@ -56,29 +129,89 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //   - luckyNumbers (array)
 //   - address (object with following key/values: number , street , state, zip)
 
+var scaryMonster = {
+  name: "Ivan the Terrible",
+  smell: "ghastly",
+  weight: 500,
+  citiesDestroyed: ['Paris', 'Miami', 'Dallas'],
+  luckyNumbers: [10, 33, 55, 75],
+};
+
+var monsterAddress = {
+  number: 1776,
+  street: "Monsters Forever Lane",
+  state: "Maryland",
+  zip: 12345
+};
+
 
 // Create a new class called SuperHero
 // - Your class should have the following DYNAMIC values
-//   - name 
+//   - name
 //   - superpower
-//   - age 
+//   - age
 // - Your class should have the following STATIC values
 //   - archNemesis, assigned to "The Syntax Error"
 //   - powerLevel = 100
-//   - energyLevel = 50 
+//   - energyLevel = 50
+
+
+class SuperHero {
+  constructor(name, superpower, age) {
+    this.name = name;
+    this.superpower = superpower;
+    this.age = age;
+    this.archNemesis = "Billy the Dog";
+    this.powerLevel = 100;
+    this.energyLevel = 50;
+  }
+sayName() {
+  console.log(this.name);
+}
+  maximizeEnergy(gainEnergy) {
+var highestPower = this.energyLevel * gainEnergy;
+this.energyLevel = highestPower;
+console.log(this.maximizeEnergy);
+}
+ gainPower(powerLevel) {
+  for (var i = 0; i < this.gainPower; i++)
```

Comment:
You set this up taking an argument, which is good. Now, you just want to add the value held in that argument to the powerLevel, and have that update. No loop needed.